client/src/pages/Newsfeed.js

```diff
@@ -7,12 +7,23 @@ import StefanNormal from '../images/FakePostImages/StefanNormal.svg';
 import SimonNormal from '../images/FakePostImages/SimonNormal.svg';
 import MolnarNormal from '../images/FakePostImages/MolnarNormal.svg';
 import styled from 'styled-components/macro';
+import { useState } from 'react';
 
 export default function Newsfeed({ open }) {
 
+    const [isVisible, setVisible] = useState(false);
+    function callbackFunction(childData) {
```

Comment:
I would use a different name for the function. callbackFunction could mean anything. Try to think what the function does.
Also the function has childData as a parameter, but never uses this parameter, so you could get rid of.